services/management/dbaas/components_service.go

```diff
@@ -244,37 +238,53 @@ func (c componentsService) CheckForOperatorUpdate(ctx context.Context, req *dbaa
 		close(responseCh)
 	}()
 
-	// Meanwhile, get latest operators version compatible with installed PMM.
 	pmmVersion, err := goversion.NewVersion(pmmversion.PMMVersion)
 	if err != nil {
 		return nil, status.Error(codes.Internal, err.Error())
 	}
-	latestPXCOperatorForInstalledPMM, latestPSMDBOperatorForInstalledPMM, err := c.versionServiceClient.GetLatestOperatorVersion(ctx, pmmVersion.Core().String())
+	resp := &dbaasv1beta1.CheckForOperatorUpdateResponse{
+		ClusterToComponents: make(map[string]*dbaasv1beta1.ComponentsUpdateInformation),
+	}
+
+	latestPXCOperatorVersion, latestPSMDBOperatorVersion, err := c.versionServiceClient.GetLatestOperatorVersion(ctx, pmmVersion.Core().String())
```

Comment:
```suggestion
	latestPXCOperatorVersion, latestPSMDBOperatorVersion, err := c.versionServiceClient.LatestOperatorsVersions(ctx, pmmVersion.Core().String())
```

Reply:
Not sure if it's right thing to do but done.